srf_generation/source_parameter_generation/uncertainties/mag_scaling.py

```diff
@@ -125,9 +125,9 @@ def a_to_mw_hanksbakum(a):
 
 def mag2mom(mw):
     """Converts magnitude to moment"""
-    return np.exp(1.5 * (mw + 10.7) * np.log(10.0))
+    return float(np.exp(3.0 / 2.0 * (mw + 10.7) * np.log(10.0)))
```

Comment:
Use np.float so we can do them in vectorisation in other scripts.

Reply:
The purpose of this function is so that academics can call it in their perturbation version script and receive a python float back, allowing it to be saved without pandas making it an incomprehensible numpy float.
A numpy version is available in createSRF.py